crates/utils/src/utils.rs

```diff
@@ -116,26 +115,28 @@ pub fn scrape_text_for_mentions(text: &str) -> Vec<MentionData> {
   out.into_iter().unique().collect()
 }
 
-pub fn is_valid_username(name: &str) -> bool {
-  VALID_USERNAME_REGEX.is_match(name)
+pub fn is_valid_actor_name(name: &str) -> bool {
+  let max_length = Settings::get()
+    .actor_name_max_length
+    .unwrap_or_else(|| Settings::default().actor_name_max_length.unwrap());
```

Comment:
You should just be able to do unwrap_or_default

Reply:
Thats not gonna work, unwrap_or_default on Option will return the default value of inner (usize), which is probably 0. The option value doesnt know about the struct it is in. I added a helper method instead. I added a helper method instead. We really need to find a good config crate so we dont have to do this manually.